site/docs/main/configuration.md

```diff
@@ -172,6 +173,16 @@ The network configuration block can be used to configure various parameters netw
 {: class="table thead-dark table-bordered"}
 <br>
 
+### Listener Configuration
+
+The listener configuration block can be used to configure various parameters for Envoy listener.
+
+| Field Name | Type| Default  | Description |
+|------------|-----|----------|-------------|
+| connection-balancer | string | `""` | This field specifies the listener connection balancer. If the value is `exact`, the listener will use the exact connection balancer. See [the Envoy documentation][14] for more information. |
```

Comment:
```suggestion
| connection-balancer | string | `""` | This field specifies the listener connection balancer. If the value is `exact`, the listener will use the exact connection balancer to balance connections between threads in a single Envoy process. See [the Envoy documentation][14] for more information. |
```

Small suggestion to make it clearer that this is Envoy-internal, and not for load-balancing to upstreams/backends.

Reply:
done it. hi @youngnick @sunjayBhatia  maybe this pr can be merged. Any other questions？